freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020  Red Hat, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from freshmaker import db
+from freshmaker.handlers import ContainerBuildHandler
+from freshmaker.events import BotasChangeStateEvent
+from freshmaker.models import Event
+from freshmaker.types import EventState
+
+
+class HandleBotasAdvisory(ContainerBuildHandler):
+    """
+    Handles event that was created by transition of an advisory filed by
+    BOTAS to SHIPPED_LIVE state
+    """
+    name = "HandleBotasAdvisory"
+    order = 0
```

Comment:
The default order seems to be `50` and it just needs to be lower than `RebuildImagesOnRPMAdvisoryChange` I believe. Can you put it a higher number but less than 50? This will give us some more room for adding handlers that need to go ahead of this handler but not having to go negative on the order.